issues.md

```diff
@@ -171,60 +165,41 @@ We need an assigned triager to regularly triage bugs coming in on each MDN conte
 
-For each bug, set a priority measure label to help people who want to work on the most important issues or areas (rather than the topics they are interested in).
+#### Set a priority label
 
-The levels of priority are:
+For each bug, set a priority label based on the severity of the issue to help people who want to work on the most important issues or areas.
 
-- `P0` — A critical issue on any MDN doc.
-- `P1` — A major issue on a Tier 1 MDN doc.
-- `P2` — A minor issue on a Tier 1 MDN doc.
-- `P3` — A major issue on a Tier 2 MDN doc.
-- `P4` — A minor issue on a Tier 2 MDN doc.
+- Critical issue: This type of issue needs to be fixed as soon as possible, regardless of where it appears on the site. This type of issue could damage MDN's reputation severely and/or harm users. Examples of this issue include an incorrect code snippet, which if used in production, could create a severe security problem and undesirable content such as malware, profanity, pornography, hate speech, or links to such content.
+  - Label: `p0` (will be addressed immediately)
 
-Definitions:
+- Major issue: This type of issue could severely affect a page's usefulness. For example, a significant amount of out-of-date information, a complex and important code example that doesn't work, a significant amount of prose that is badly written and hard to understand, or a large number of broken links.
+  - Labels: `p1` (will be addressed soon) and `p2` (will be addressed soon, but higher priority items will take precedence)
 
-- Critical issue — Something that could damage MDN's reputation severely and/or harm users, which we need to fix as soon as possible, regardless of where it appears on the site. Examples include code examples that if used in production could create a severe security issue, undesirable content such as malware, profanity, pornography, hate speech, or other undesirable content, or links to such content.
-- Major issue — Something that could severely affect a page's usefulness, for example a significant amount of out-of-date information, a complex and important code example that doesn't work, a significant amount of prose that is badly written and hard to understand, a large number of broken links, etc.
-- Minor issue — something that doesn't look great but does not affect learning, or only has a minor effect on learning. Examples — Typos, bad grammar, a broken link, a small amount of out-of-date information or badly-written prose, a small code snippet that doesn't work.
+- Minor issue — This is a type of improvement issue that can make the existing content better but does not affect learning or only has a minor effect on learning. Examples include typos, bad grammar, a broken link, a small amount of out-of-date information or badly-written prose, or a code snippet that doesn't work.
+  - Labels: `p3` (no visibility when the issue will be addressed)
```

Comment:
Should we mention that community help here is appreciated?

Reply:
Fixed in https://github.com/mdn/temp-processes/pull/7/commits/89f1dce3e897d9f1c11704030f372f65f1f2ea94, line 182.